### ISSUE TITLE
[BUG] Keyboard focus on entering and leaving checker help #3161

### ISSUE THREAD

**Issue by aliunwala** [IBMa/equal-access#945]
## What tool(s) are you using?

- [ ] `able site`
- [ ] `accessibility-checker-engine`
- [x] `accessibility-checker`
- [ ] `karma-accessibility-checker`
- [ ] `rule-server`
- [ ] `ACS`                              
- [ ] `boilerplates`
- [ ] `other`

## Detailed description

> Platform: Mac, Chrome

> When using the keyboard to navigate the Accessibility Checker, when I select 'Learn more' to open the help for any issue, the keyboard focus jumps out of the checker tab to the DOM element, instead of going into the help.  Then when I close the help, the keyboard focus is returned to the top of the checker tab instead of the issue I was exploring.

## Steps to reproduce the issue

1. Open the Accessibility Checker tab in the Elements panel
2. Scan any page (e.g. [Altoro Mutual](https://demo.testfire.net))
3. Using the keyboard, tab to the first issue's 'Learn more' link
4. Press 'Enter' to activate the link
5. Observe where keyboard focus is - on the selected element in the DOM. It should be on the first link in the help.
6. Use the tab key to navigate into the Checker tab to the 'Back to list view' button
7. Press Enter to activate the button
8. Observe where the keyboard focus is - at the top of the checker tab. It should be back on the original issue that you selected

## Additional information

![Screen Shot 2021-06-01 at 4.15.25 PM.png](https://zenhub.ibm.com/images/5821294ffdfd5b9223bb7e4d/7129625b-415f-4480-8db8-b002c9580b98)

![Screen Shot 2021-06-01 at 4.14.15 PM.png](https://zenhub.ibm.com/images/5821294ffdfd5b9223bb7e4d/3882a808-5a13-4462-9f92-e93a40ddfe38)

![Screen Shot 2021-06-01 at 4.13.58 PM.png](https://zenhub.ibm.com/images/5821294ffdfd5b9223bb7e4d/5d9b0ed0-d1e6-47a2-8766-4b96e9b5bc26)

![image](https://user-images.githubusercontent.com/4451128/172455612-9db30a1b-08cf-4306-a2d6-b6db0cb160f4.png)

**Comment by drjoho** [comment]
Have fixed 4 of 5 problems here:
1. When open checker focus on "Scan" button
2. When go to learn more the focus is on "Back to list view" button
3. When return from learn more the focus is on "Learn More" line (NOT DONE YET)
4. When got to view stored scans the focus in on "Back to list view" button
5. When return from view stored scans the focus on "Scan" button

**Comment by drjoho** [comment]
Done.

**Comment by eirckr** [comment]
Validated, the focus is working as expected. For the # 5 Observe where keyboard focus is - on the selected element in the DOM. It should be on the first link in the help. The first link is the Back to list view.
